incubator/kubevirt/README.md

```diff
@@ -0,0 +1,61 @@
+# KubeVirt Helm Chart
+It will deploy [KubeVirt](https://github.com/kubevirt/kubevirt) resources to make possible to create VM's as part of k8s environment.
+You can find more information about VM yaml specification under [KubeVirt User Guide](https://kubevirt.gitbooks.io/user-guide/)
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
+```
+
+You will need to give additional permissions to the user `system:serviceaccount:kube-system:default`, it will be valid only for cases when you initiated `helm` under `kube-system` namespace.
+
+```
+$ kubectl create clusterrolebinding --user system:serviceaccount:kube-system:default kube-system-cluster-admin --clusterrole cluster-admin
+```
+
+You will need to specify your master IP under `master.ip` variable and master hostname under `master.selector` variable, because most components must run on the same node and will talk via external IP(I hope it will change soon).
+```
+$ helm install --name my-release incubator/kubevirt --set master.ip=10.10.10.10,master.selector=master.test.com
+```
+
+After installation finish, it better to remove extra permissions from the system.
+
+```
+$ kubectl delete clusterrolebinding kube-system-cluster-admin
+```
+
+## Deleting the Charts
+
+Delete the Helm deployment as normal
+
+```
+$ helm delete my-release
+```
+
+## Configuration
+
+The following tables lists the configurable parameters of the kubevirt chart and their default values.
+
+|                Parameter                  |            Description              |          Default           |
+| ----------------------------------------- | ------------------------------------| -------------------------- |
+| `master.ip`                               |k8s master IP                        | `10.10.10.10`              |
+| `master.hostname`                         |k8s master hostname                  | `master.test.com`          |
```

Comment:
master.hostname is not needed here, since our components don't have placing requirements per se. However treating them as critical add-on components like mentioned here: https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/ would something I consider to be the correct way to go. Note that you don't necessarily want to treat the squid spice proxy as a critical add-on. It is not vital for KubeVirt and can have a lot of traffic.